examples/getting_started/run.py

```diff
@@ -31,6 +31,12 @@
     # load config
     config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
 
+    # modify config values
```

Comment:
What is the motivation for doing this modification in the code? We don't usually do this for other config values (unless there is a good reason, like they're being modified by a command-line argument or something). Why start here?

Reply:
Aah, you are right. I'm not sure why I did that, instead of having these updates in the yaml file. I updated this.